types/swift/AnalyticsEvent.swift

```diff
@@ -0,0 +1,36 @@
+// 
```

Comment:
Should we delete the original one from `base_types` or would that break Element? Might be worth importing the swift package into Element too.

Reply:
Well the main issue here is that the stub generator deletes `types/swift` before generating the latest types. So for now it will need to be a copy each time. And yep, we definitely should migrate to the package for EI as we'll be able to specify a version instead of a commit on the `release/swift` branch.